doc/gov/chapters/chapter02.md

```diff
@@ -1,11 +1,11 @@
 [<< Back](../)
 
 # 2. Onboarding
-<p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
+<p align="right"><img src="../figures/bogo_sdc.png" alt="scope" title="Scope" width="35%"/></p>
 
 ## Table of Contents
 * [2.1 Introduction](#2.1)
-* [2.2 Membership](#2.2)
+* [2.2 Request Membership](#2.2)
```

Comment:
Is it possible to request a membership to a task force?

Reply:
Gergely, technically yes. "task force" by definition really just means the org was assembled to make a special thing/'task' happen, and it typically has a short-term connotation. But it's still an organization. But I take your point as we need some introductory language that says anyone/org is free to join because it is an open source community...etc...so folks don't get confused.